company.el

```diff
@@ -2521,11 +2540,18 @@ inserted."
       (when company-candidates
         (setq this-command 'company-complete-common)))))
 
-(defun company-complete-number (n)
+(define-obsolete-function-alias
+ 'company-complete-number
+ 'company-complete-quick-access
+ "0.10.0")
+
+(defun company-complete-quick-access (n)
```

Comment:
Should we perhaps keep this command's name?

It only works with numbers: if the user wants to handle different inputs, they will need to either change these bindings, or add key translations as you described. But this command will still only work with numbers itself.

Reply:
Yes, I understand the concerns.
I gave it a lot of thought and  concluded that the 'quick-access' part:
  - better represents what the function does (with a high chance of `company-complete-number` name giving an impression that's a function to complete numbers a user types).
  - provides better uniformity for quick-access functionality throughout the code.

Also, I tried and failed to use it interactively, so might be that's where an important detail could be for keeping the 'number' (or 'digit') part?

Anyway, **I suggest postponing with the final decision and get back to this question on the next review.**

Reply:
By "to complete numbers a user types" I mean typing in the editor before tooltip shows-up (kind of an alternate function to complete only numbers.)

Reply:
> By "to complete numbers a user types" I mean typing in the editor before tooltip shows-up (kind of an alternate function to complete only numbers.)

Yeah, I get what you mean. But perhaps there could be another name which is also as specific (e.g. probably has "number" in it), but doesn't give that impression.

> I suggest postponing with the final decision and get back to this question on the next review

Sounds good. My objection wasn't strong anyway.

Reply:
Yeah, I've started considering a rather longish `company-complete-quick-access-(digit|number)`. Will give it some more thought.